R/pkg/R/sparkR.R

```diff
@@ -328,6 +328,7 @@ sparkRHive.init <- function(jsc = NULL) {
 #' @param sparkPackages Character vector of packages from spark-packages.org
 #' @param enableHiveSupport Enable support for Hive, fallback if not built with Hive support; once
 #'        set, this cannot be turned off on an existing session
+#' @param ... additional parameters passed to the method
```

Comment:
I'd clarify as in L 317, for example, "named Spark properties passed to the method"